docs/src/configuration.md

```diff
@@ -81,35 +102,59 @@ The following implementations should work:
 - [Microsoft MPI](https://docs.microsoft.com/en-us/message-passing-interface/microsoft-mpi)
 - [IBM Spectrum MPI](https://www.ibm.com/us-en/marketplace/spectrum-mpi)
 
-If the implementation is changed, you will need to re-run `Pkg.build("MPI")`.
+#### Advanced options
 
-### [Environment variables](@id environment_variables)
+```@doc
+MPI.use_system_binary
+```
+You can use the argument `mpiexec` to provide the the name (or full path) of the MPI launcher executable. The default is
+`mpiexec`, but some clusters require using the scheduler launcher interface (e.g. `srun`
+on Slurm, `aprun` on PBS). If the MPI library has an uncommon name you can provide it in `library_names`.
+In case ABI detection fails you can provide a manual choice,
+but also open an issue so that the auto-detection can be improved. `export_prefs=true` can be used to copy the preferences into the `Project.toml` instead of creating a
+`LocalPreferences.toml` file to hold them.
+
+#### Notes to HPC cluster adminstators
+
+Preferences are merged across the Julia load path, so it is feasible to provide a module file that appends a path to
+`JULIA_LOAD_PATH` variable that contains system wide preferences.
 
-The following optional environment variables can be used to control certain aspects of the
-build script and other library behaviour. The results of these will be cached in a
-configuration file located at `~/.julia/prefs/MPI.toml` and so can be used for subsequent
-MPI builds.
+As an example you can use [`MPI.use_system_binary(;export_prefs)`](@ref) to create a file `Project.toml` containing:
```

Comment:
Why the `export_prefs`? Why can't I just copy the `LocalPreferences.toml` to the central location? If I have to use `export_prefs=true`, it should be stated explicitly as such.

Reply:
I only wanted to show one file instead of two, but you are probably right. I personally prefer the LocalPreferences.toml

Reply:
If `LocalPreferences.toml` works (and can also put on the load path), and if everything else is all the same, I'd probably just describe this approach. Users familiar with Preferences.jl will know that the other option exists, while users who don't are not overwhelmed by more choices.

Another advantage of `LocalPreferences.jl` is that it is imho *much* less ugly if you have a local project in a git repo, since you can just add `LocalPreferences.jl` to `.gitignore`, while that is not possible for `Project.toml`.

Thus I suggest to reformulate this as more "do this" instead of "this is one way of doing it" :-)